modules/51_internalizeDamages/KWlikeItrTest/realization.gms

```diff
@@ -0,0 +1,6 @@
+
```

Comment:
description would be appreciated. Do you really want to merge the `Test` realizations as well?

Reply:
I renamed it from test to KW_SE